web/src/components/nomination/AffiliationInput.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-row dense>
+    <v-col
+      cols="12"
+      md="6"
+      xs="12"
+    >
+      <v-text-field
+        ref="title"
```

Comment:
I wonder if vue will mind if there are multiple refs with the same name on the rendered page, or if keeps track by component.

Reply:
I'll change it just in case :)